phidgets_analog_outputs/CHANGELOG.rst

```diff
@@ -0,0 +1,14 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package phidgets_analog_outputs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.0.5 (2022-02-17)
+------------------
+
+1.0.4 (2021-10-22)
+------------------
+
+1.0.3 (2021-09-29)
+------------------
+* Add Analog Outputs (`#103 <https://github.com/ros-drivers/phidgets_drivers/issues/103>`_)
+* Contributors: Carsten Plasberg
```

Comment:
Let's remove this file for now.  We'll automatically generate it when we do a new release with this package in place.